src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -7255,14 +7255,14 @@ s32 Player_HandleSlopes(PlayState* play, Player* this, CollisionPoly* floorPoly)
             this->pushedYaw = downwardSlopeYaw;
             Math_StepToF(&this->pushedSpeed, slopeSlowdownSpeed, slopeSlowdownSpeedStep);
         } else {
-            // moving downward on the slope, causing player to slip
-            Player_SetupAction(play, this, Player_Action_8084F390, 0);
+            // moving downward on the slope, causing player to slide
+            Player_SetupAction(play, this, Player_Action_SlideOnSlope, 0);
             func_80832564(play, this);
 
             if (sFloorShapePitch >= 0) {
-                this->av1.actionVar1 = 1;
+                this->av1.slideFacingUpSlope = true;
             }
```

Comment:
I think just `facingUpSlope` is fine here and slide doesnt necessarily need to be in the name, this much is clear from context

Reply:
Done.